### PR TITLE
GITHUB-5949 - Allow to delete import/export profile on the profile page

### PR DESCRIPTION
**Pull request by BatsaxIV**
[//]: <> (<3 Thanks for taking the time to contribute! You're awesome! <3)

[//]: <> (If you've never contributed to this repository before, please read https://github.com/akeneo/pim-community-dev/blob/master/.github/CONTRIBUTING.md)

**Description (for Contributor and Core Developer)**
Fix https://github.com/akeneo/pim-community-dev/issues/5949 - Allow to delete import/export profile on the profile page

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | -
| Added Behats                      | Ok
| Added integration tests           | -
| Changelog updated                 | Ok
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | -
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

**Comment by SamirBoulil** [comment]
Hello @BatsaxIV,

I was wondering if you needed any help regarding this PR ? It would be cool to have it merged !

**Comment by SamirBoulil** [comment]
Hello @BatsaxIV,

Today I've pushed the missing behats as well as the updated changelog in a separate commit.

Let me know if everything is good with you!

File: src/Pim/Bundle/EnrichBundle/Resources/translations/jsmessages.en.yml [hunk 1]
```diff
@@ -14,6 +14,7 @@ confirmation:
         export profile:   Are you sure you want to delete this export profile?
         user:             Are you sure you want to delete this user?
         role:             Are you sure you want to delete this role?
+        job_instance:     Are you sure you want to delete this job instance ?
```
Comment (BatsaxIV) [inline src/Pim/Bundle/EnrichBundle/Resources/translations/jsmessages.en.yml]: There is no space between **instance** and **?**

**Comment by BatsaxIV** [comment]
Hi @SamirBoulil,

Thanks for finishing the job, I'll check the test (for my personnal culture on akeneo) andlet you know if everything is fine for me!

**Comment by SamirBoulil** [comment]
Ok, in the meantime I'll run the CI on it, have it reviewed by the team ^^

File: CHANGELOG-1.7.md [hunk 2]
```diff
@@ -17,6 +17,7 @@
 - PIM-6283: Fix a bug where SKUs of products in the Variant Group edit page were not displayed
 - PIM-6274: Successfully validate products with a custom validation on identifier
 - PIM-6199: Fix product mass edit attribute add select clickable on confirmation page
+- GITHUB-6105: Fix the deletion of a job instance (import/export) from the job edit page
```
Comment (BatsaxIV) [inline CHANGELOG-1.7.md]: Should'nt we reference the issue GITHUB-5949 ? (like here : https://github.com/akeneo/pim-community-dev/pull/6081/files)

Comment (a2xchip) [inline CHANGELOG-1.7.md]: it should be GH issue number or PR num if no issue exists (your version). The 5949 issue exists, so you should refer to it in CHANGELOG.

**Comment by SamirBoulil** [comment]
Congratulations @BatsaxIV, Your PR has been merged !

🎉  Again, thank you for your contribution ! 

You can now claim your badge "Core contributor" on http://badger.akeneo.com if you don't already have it. ^^

If you wish to contribute again, don't hesitate to look at our github project board where we labeled all kinds of tasks to take on ! https://github.com/akeneo/pim-community-dev/projects/1

**Comment by a2xchip** [comment]
Congrats @BatsaxIV :tada:

**Pull request merged**